amundsen_gremlin/neptune_bulk_loader/api.py

```diff
@@ -24,6 +24,7 @@
 )
 from gremlin_python.process.anonymous_traversal import traversal
 from gremlin_python.process.graph_traversal import GraphTraversalSource
+from neptune_python_utils.endpoints import Endpoints, RequestParameters
```

Comment:
Due to the wacky way we add neptune_python_utils as a submodule here, if you forget to setup.py install as I apparently did last time, isort will think it's part of this package. It belongs up here.